doc/index.html.en

```diff
@@ -5,7 +5,7 @@ lang_suffix: true
 no_sidebar: true
 ---
 <div class="latest-release">
-  <p><a href="/en/news.html">{{ site.version }}</a> is the latest release. It had been released at {{ site.release_date }}.</p>
```

Comment:
> Thanks for reporting this.
> 
> I'm not a native English speaker. So I don't know the different of "has been released" and "was released".
> 
> I often see "XXX has been released" announce. Why should we use "was released" here?

Then I think we agree on that "had been released" is wrong ;)

Reply:
Ah, "had" is wrong. It's a typo.